### PR TITLE
fix some tests for split build

### PR DESCRIPTION
**Pull request by akuzm**
Changelog category (leave one):
- Not for changelog (changelog entry is not required)

### CODE REVIEW

**Comment by akuzm** [comment]
Both debug build and split build slow it down:

- release -- no more than 36 minutes
- release split -- 37 minutes
- debug -Og split -- 42 minutes

I'll merge the fixes for tests in split build.

File: docker/test/fasttest/run.sh [hunk 1]
```diff
@@ -259,7 +268,7 @@ TESTS_TO_SKIP=(
     00974_query_profiler
 
     # Look at DistributedFilesToInsert, so cannot run in parallel.
-    01460_DistributedFilesToInsert
+    01457_DistributedFilesToInsert
```
Comment (azat) [inline docker/test/fasttest/run.sh:271]: But the test name is `01460_DistributedFilesToInsert`, why did you change this?

Comment (akuzm) [reply docker/test/fasttest/run.sh:271]: This is some weird mistake, thanks for noticing, I'll fix.
